src/juxt/mail/alpha/mail.clj

```diff
@@ -72,15 +75,27 @@
       "<blank>"))))
 
 (defmethod triggers/run-action! ::mail/send-emails
-  [{::site/keys [db ses-client]} {:keys [trigger action-data]}]
-  (let [{::mail/keys [html-template text-template from subject]} trigger
+  [{::site/keys [db smtp-client]} {:keys [trigger action-data]}]
+  (let [{::mail/keys [html-template
+                      text-template
+                      from
+                      subject
+                      host
+                      username
+                      password
+                      port]} trigger
```

Comment:
Triggers are stateful entries in the DB. We try to keep them in sync with [this file](https://github.com/juxt/rlws/blob/master/rlws.iot-cloud-site/resources/service-resources.edn#L697)

The trigger map can't return SMTP-client specific data such as `username`, `password`, etc. These params should ideally be configured by the environment, eg through the integrant configuration or as env variables.